tests/component-tests/tooltip/js/TooltipTests.js

```diff
@@ -194,10 +194,107 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         }
     });
 
+    // FLUID-5846 tests for when tooltip is run on content in an iframe
+
+    fluid.defaults("fluid.tests.tooltip.FLUID5846", {
+        gradeNames: ["fluid.tooltip"],
+        selectors: {
+            item1: "#tooltip-item-1",
+            item2: "#tooltip-item-2"
+        },
+        model: {
+            idToContent: {
+                "tooltip-item-1": "item 1",
+                "tooltip-item-2": "item 2"
+            }
+        }
+
+    });
+
+    fluid.tests.tooltip.FLUID5846.setupIframe = function (that, iframe) {
+        $(iframe).load(function () {
+            // DO NOT MOVE this property access outside this function!
+            var dokkument = iframe.contentDocument;
+            var iframeWindow = dokkument.defaultView;
+            var iframejQuery = iframeWindow.jQuery;
+            that.iframeBody = iframejQuery("body", dokkument);
+            that.events.iframeReady.fire();
+        });
+    };
+
+    fluid.defaults("fluid.tests.tooltip.FLUID5846.parent", {
+        gradeNames: ["fluid.viewComponent"],
+        selectors: {
+            iframe: ".FLUID-5846-iframe"
+        },
+        events: {
+            iframeReady: null,
+            onReady: null,
+            afterOpen: null,
+            afterClose: null
+        },
+        listeners: {
+            "onCreate.setupIframe": "fluid.tests.tooltip.FLUID5846.setupIframe({that}, {that}.dom.iframe.0)"
```

Comment:
Should `{that}.dom.iframe.0` just be `{that}.dom.iframe` as the function seems to need to use it as a jQuery element anyways.